config/pipeline.yaml

```diff
@@ -447,6 +447,20 @@ jobs:
         patchlevel: 7
     kcidb_test_suite: kselftest.dt
 
+  kselftest-discoverable-devices-probe:
+    template: kselftest.jinja2
```

Comment:
Depending on when this PR gets merged, be advised that the job template might need some adjustment; see https://github.com/kernelci/kernelci-pipeline/pull/655

Reply:
This job template no longer exists. It was replaced by [`generic.jinja2`](https://github.com/kernelci/kernelci-pipeline/blob/main/config/runtime/generic.jinja2) - see [the example](https://github.com/kernelci/kernelci-pipeline/blob/main/doc/developer-documentation.md#an-example-of-enabling-a-new-job) for more details.

Reply:
Updated.